### ISSUE TITLE
Compilation error line numbers do not account for multi-line strings

### ISSUE THREAD

**Issue by AndrewAday** [ccrma/chuck#457]
```c
"
multi
line
chuck
string
" => string foo;

[1,2,3] => int bar[]; // syntax error
```

Chuck vm reports: 
```
playground.ck:3:9: error: cannot resolve operator '=>' on types 'int[]' and 'int[]'...
[3] line
            ^
playground.ck:3:9: error: ...(note: use '@=>' for object reference assignment)
[3] line
```

The reported line number of the syntax error "3" is correct only if the string foo is not multi-line.

**Comment by gewang** [comment]
doh! looking into it hah

**Comment by gewang** [comment]
this issue has been addressed and has been incorporated into chuck-1.5.3.1
